appinfo/routes.php

```diff
@@ -27,12 +27,28 @@
 	'ocs' => [
 		[
 			'name' => 'API#generate',
-			'url' => '/api/v1/generate',
+			'url' => '/api/{apiVersion}/generate',
 			'verb' => 'GET',
+			'requirements' => [
+				'apiVersion' => 'v(1|1\.1)',
+			],
 		],
 		[
 			'name' => 'API#validate',
-			'url' => '/api/v1/validate',
+			'url' => '/api/{apiVersion}/validate',
+			'verb' => 'POST',
+			'requirements' => [
+				'apiVersion' => 'v(1|1\.1)',
+			],
+		],
+		[
+			'name' => 'API#generateShare',
+			'url' => '/api/v1.1/share/generate',
```

Comment:
all other apps just bump it as an integer atm, so would go with v2

Reply:
Done 👍